Framework/LiveData/src/Kafka/private/Schema/ai33_det_count_imgs_generated.h

```diff
@@ -0,0 +1,111 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Agreed that this should not be modified, but how does one find the source schema for this type @MikeHart85 ?

Reply:
Schemas are at https://github.com/ess-dmsc/streaming-data-types/
maybe this is a good place to document that.